docs/collecting-data/collecting-from-own-applications/rust-tracker/index.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "Rust Tracker"
+date: "2022-10-24"
+sidebar_position: 310
```

Comment:
I think we could move the tracker a bit higher in the sidebar somewhere around the server-side trackers, maybe after PHP?